setup.py

```diff
@@ -1,12 +1,16 @@
 from setuptools import setup
 
+#bring in __version__ from source
+#per https://stackoverflow.com/a/17626524
+#and https://stackoverflow.com/a/2073599
+with open('lineman/version.py') as ver: exec(ver.read())
```

Comment:
So I read something about race conditions when I was looking at these SO links. 

I didnt know you could scope like that in python without a line break after the colon. Could you change that? It is maybe hard to read